### PR TITLE
Billing check bank details reuse + beautify payment method modal

### PR DESCRIPTION
**Pull request by cabch**
- added stripe text
- added checkbox for bank details reuse

- changed modal ui
- reorganized folders to separate global payment stuff from stripe specific
- added locale error messages
- handled save + exit button to avoid multiple clicking

### CODE REVIEW

File: src/assets/i18n/fr.json [hunk 1]
```diff
@@ -1513,6 +1513,13 @@
       "payment_methods_status": "Status",
       "payment_methods_type": "Type",
       "payment_methods_brand": "Marque",
+      "payment_methods_card": "Carte",
+      "payment_methods_verification_code": "Code de vérification",
+      "payment_methods_conditions": "Pour des raisons de sécurité, nous ne stockons ni ne traitons vos informations de moyen de paiement. Les informations collectées sont envoyées de manière sécurisée à notre partenaire STRIPE qui fourni l'infrastructure de facturation et de paiement.",
+      "payment_methods_check": "J'ai lu les conditions générales d'utilisation et j'accepte que mes informations de paiement soit utilisées pour payer mes futures sessions de recharge.",
```
Comment (ClaudeROSSI) [inline src/assets/i18n/fr.json:1519]: soit ==> soient!

File: src/assets/i18n/fr.json [hunk 2]
```diff
@@ -1513,6 +1513,13 @@
       "payment_methods_status": "Status",
       "payment_methods_type": "Type",
       "payment_methods_brand": "Marque",
+      "payment_methods_card": "Carte",
+      "payment_methods_verification_code": "Code de vérification",
+      "payment_methods_conditions": "Pour des raisons de sécurité, nous ne stockons ni ne traitons vos informations de moyen de paiement. Les informations collectées sont envoyées de manière sécurisée à notre partenaire STRIPE qui fourni l'infrastructure de facturation et de paiement.",
```
Comment (ClaudeROSSI) [inline src/assets/i18n/fr.json:1518]: fourni ==> fournit

File: src/assets/scss/material-dashboard.scss [hunk 3]
```diff
@@ -54,6 +54,8 @@
 
 @import "views/assets/assets";
 
+@import "views/billing/\ billing.scss";
```
Comment (ClaudeROSSI) [inline src/assets/scss/material-dashboard.scss:57]: c'est normal le /\ ???

File: src/assets/scss/views/billing/ billing.scss [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+  background: rgba($brand-primary, 0.3);
+  font-style: italic;
+}
+
+.mat-payment-error {
+  color: $brand-danger;
+  font-size: 0.8em;
+  margin-top: -.8em;
+  text-align: left;
+}
```
Comment (ClaudeROSSI) [inline src/assets/scss/views/billing/ billing.scss:15]: file name starts with a space!!!! ????

**Review by ClaudeROSSI — CHANGES_REQUESTED** [review]
3 minor issues to fix!

File: src/assets/scss/material-dashboard.scss (hunk 3 above)
Comment (cabch) [reply src/assets/scss/material-dashboard.scss]: non c'était à cause de l'espace du nom du fichier. Done ✅

File: src/assets/scss/views/billing/ billing.scss (hunk 4 above)
Comment (cabch) [reply src/assets/scss/views/billing/ billing.scss]: oups
changed ✅

File: src/assets/i18n/fr.json (hunk 2 above)
Comment (cabch) [reply src/assets/i18n/fr.json]: done ✅

File: src/assets/i18n/fr.json (hunk 1 above)
Comment (cabch) [reply src/assets/i18n/fr.json]: done ✅

**Review by ClaudeROSSI — APPROVED** [review]
ok

**Pull request merged**